go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/operator-framework/operator-sdk v0.17.0
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/rook/rook v1.4.6
+	github.com/rook/rook v1.5.1
```

Comment:
FYI the update of rook to v1.5.1 is now covered in a separate PR #912

Reply:
Thanks @obnoxxx , once the initial testing is complete, will re-base my PR on top of that...

Reply:
PR rebased on top of Umanga's rook updation PR